src/main/java/com/google/cloud/pontem/CloudSpannerDatabaseBackup.java

```diff
@@ -120,7 +119,7 @@
  * </pre>
  */
 public class CloudSpannerDatabaseBackup {
-  private static final Logger LOG = LoggerFactory.getLogger(CloudSpannerDatabaseBackup.class);
+  private static final Logger LOG = Logger.getLogger(CloudSpannerDatabaseBackup.class.getName());
```

Comment:
Note even though it's static and final the logger isn't considered a constant and therefore, you can avoid the uppercase naming. I'll leave it up to you whether you want to change it or not.